zerver/signals.py

```diff
@@ -13,51 +13,32 @@
 from zerver.lib.send_email import FromAddress
 from zerver.models import UserProfile
 from zerver.lib.timezone import get_timezone
+from ua_parser import user_agent_parser
 
 JUST_CREATED_THRESHOLD = 60
 
 def get_device_browser(user_agent: str) -> Optional[str]:
-    user_agent = user_agent.lower()
-    if "zulip" in user_agent:
+    device_browser = user_agent_parser.ParseUserAgent(user_agent)['family']
+    if "zulip" in user_agent.lower():
```

Comment:
Could be `user_agent.startswith("Zulip")`, and before the `ParseUserAgent` call.

Reply:
Yeah, that's better. changed.